src/olmo_core/data/mixes/__init__.py

```diff
@@ -7,18 +7,14 @@
 
 from ..tokenizer import TokenizerName
 
-__all__ = ["DataMix"]
+__all__ = ["BaseDataMix", "DataMix"]
 
 
-class DataMix(StrEnum):
+class BaseDataMix(StrEnum):
```

Comment:
Oh, minor nit, but I've gone with the convention of calling base classes `XXXBase` instead of `BaseXXX`. Can you change this to `DataMixBase`?

Reply:
Done